.spelling

```diff
@@ -591,6 +591,7 @@ linters
 list_by_publishdate
 Liu
 liveness
+LoadBalancer
```

Comment:
I don't think you need this (but I'm totally OK with it being there)

All else LGTM.

Reply:
I could just change the doc to "load balancer", but I wanted to use the actual k8s name. Alternatively, I could escape it with back-ticks instead, but because it's used in link text, the link then displays as a mix of blue and red text, which I don't like the look of, so I opted for adding it to .spelling instead.